keys/keys.go

```diff
@@ -21,8 +21,10 @@ package keys
 import (
 	"bytes"
 	"fmt"
+	"time"
 
 	"github.com/cockroachdb/cockroach/roachpb"
+	"github.com/cockroachdb/cockroach/sql/parser"
```

Comment:
This is an awkward import to add. I think it can be avoided by avoiding usage of the `parser.Datum` types in `decodeKeyPrint`. They aren't doing much.